addons/validate_mission/fn_run.sqf

```diff
@@ -156,6 +156,72 @@ private _check_cba_settings = {
     ["Changed CBA settings", true, [], _strings];
 };
 
+private _check_playable_units = {
+    private _msg = [];
+    private _pass = true;
+    if (count playableUnits < 35) then {
+        (_msg pushBack "Not enough playable units: 35 required")};
+        _pass = false;
```

Comment:
Sneaky. The useless `(` and `)` actually hide the problem so it's hard to notice.